DIPs/DIP-5 Keycard.md

```diff
@@ -0,0 +1,78 @@
+---
+Title: Keycard at Devcon
+Status: Draft
+Primary Contact: guylouis@status.im
+Themes: Ticketing, Purchase & ID
+Tags: Attendee Experience
+Resources Required: Operational support, Communication support
+Discussions-to: https://forum.devcon.org/t/dip-5-keycard-at-devcon/46
+---
+
+## Summary
+
+Status wants to offer to Devcon attendees a keycard (https://get.keycard.tech/) coming preloaded with crypto assets (e.g NFTs and ERC-20). It can be used at Devcon to easily (at the tap of a card) collect more assets, prove ownership of them, and transfer them. 
+
+## Abstract
+
+Keycard is an open source implementation to enable smartcards in the crypto space. The framework allows to use smartcards both as highly secure hardware wallets with one's phone, and as super simple to use (think: one tap) devices that can be tapped on someone else's phone to sign a transaction, opening up a variety of use cases.
+
+A Devcon attendee (using Android) would be handed over a keycard when checking in at Devcon. This keycard will be printed with some indication of which assets are offered with the card. The user then can:
```

Comment:
We have mentioned for android only because this is what is working as of now 08/2020. If you think we need to adress android + iOS users for devcon, we can do it if the DiP is selected. Any idea when Devcon will happen in 2021 ?

Reply:
Unfortunately we cannot yet share it publicly - I wrote you a mail about it just now

Reply:
We can discuss the android vs ios compatibility after a Devcon team review. For now, this passes the editor's review and is mergeable! Still with the status as draft.